test/realizations/catchments/Bmi_C_Adapter_Test.cpp

```diff
@@ -9,14 +9,16 @@
 
 #ifndef BMI_CFE_LOCAL_LIB_NAME
 #ifdef __APPLE__
-#define BMI_CFE_LOCAL_LIB_NAME "libcfemodel.dylib"
+#define BMI_CFE_LOCAL_LIB_NAME "libcfebmi.dylib"
```

Comment:
In this cases it's because the configurations are part of the code.   The realization configurations used in these tests are not written in a separate file.  I can't remember entirely why I did those through the setup function and not from files.

There definitely does need to be some examination of how to improve some of the test design.

Reply:
For some reason I thought this was in the Adapter, not in the Test.  Not as much of a worry, but something to think about.